ckanext/canada/templates/home/index.html

```diff
@@ -0,0 +1,7 @@
+{% ckan_extends %}
+
+{% if not g.is_registry %}
+  {# Portal #}
+  {# hide CKAN front page on public site #}
+  {{ h.redirect_to('dataset.search') }}
```

Comment:
I don't think this works with flask does it?

Reply:
No it does not, we have this logic in the views now I think for:
```
@canada_views.route('/', methods=['GET'])
def home():
```
and
```
@canada_views.route('/links', methods=['GET'])
def links():
```
so we can remove that for sure.